_studio/mfx_lib/encode_hw/h264/include/mfx_h264_encode_hw_utils.h

```diff
@@ -265,7 +265,8 @@ namespace MfxHwH264Encode
 
     void PrepareSeiMessage(
         MfxVideoParam const &   par,
-        mfxExtAvcSeiRecPoint &  msg);
+        mfxExtAvcSeiRecPoint &  msg,
+        mfxU16 rec_frame_cnt = 0);
```

Comment:
Usually 'rec' stands for reconstruct. Thus better rename to ~ 'recovery' to avoid confuses.

Reply:
Can we avoid argument by default?

Reply:
Done